language-configuration.json

```diff
@@ -0,0 +1,39 @@
+{
+  "comments": {
+    "lineComment": ["{% # ", "{%}"],
```

Comment:
This is not yet implemented. Will be once https://github.com/Shopify/liquid-c/pull/144 is deployed.